css-display-4/Overview.bs

```diff
@@ -1149,10 +1149,22 @@ Reading Order: the 'reading-order' property</h3>
 	The 'reading-order' property controls the order in which
 	elements are rendered to speech
 	or are navigated to when using (linear) sequention navigation methods.
-	It takes a single <dfn value for=reading-order><<integer>></dfn> value,
-	which specifies which ordinal group the item belongs to.
-	Sibling elements are ordered starting from the lowest numbered ordinal group and going up;
-	elements with the same ordinal group are keep the order they appear in the source document.
+	This is their <dfn>reading and navigation order</dfn>.
+
+	<dl dfn-type=value dfn-for=reading-order>
+		<dt><dfn>auto</dfn>
+		<dd>
+			Same as ''0'' except that in randomizing formatting contexts
+			(which are only reasonable to apply to inherently unordered content)
+			the reading order is matched to the layout order.
```

Comment:
This doesn't explain the relative order of items with `auto` and `0`.

Reply:
That's what the cross-reference on the next line is for...?